server/index.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs');
+const https = require('https');
+const app = require('./app');
+
+const privateKey = fs.readFileSync('./security/server.key');
+const certificate = fs.readFileSync('./security/server.crt');
+
+https.createServer({
+  key: privateKey,
+  cert: certificate,
+}, app).listen(app.get('PORT'), () => {
+  console.log('App running on port', app.get('PORT'));
```

Comment:
There is a way to disable eslint inline when you want to ignore a warning in a specific case. In this case, you do want to keep the `no-console` rule (because it's true that in a vast majority of cases, we don't want console logs in our production environment) but it's ok to disable it just this time to allow you to console log that your app is running. The way to do this is like this:

```
https.createServer({
  key: privateKey,
  cert: certificate,
}, app).listen(app.get('PORT'), () => {
  // eslint-disable-next-line no-console
  console.log('App running on port', app.get('PORT'));
});   
```
Or, if you want to disable the warning on the same line:

```
https.createServer({
  key: privateKey,
  cert: certificate,
}, app).listen(app.get('PORT'), () => {
  console.log('App running on port', app.get('PORT'));  // eslint-disable-line no-console
});   
```

All rules can be disabled inline like this, but of course if you find yourself always disabling the same rule then maybe it is worth revisiting the rule configuration. Hope you find this little piece of information helpful ☺️

Reply:
Eslint is also showing errors in the `test.js` file, please fix (or configure 😉 )